src/schemas.py

```diff
@@ -59,79 +59,109 @@ class Distribution(BaseModel):
     media_type: str
     table_schema: TableSchema
 
-
-# If we wanted to provide the ability to attach arbitrary RDF we might want to
-# look at https://github.com/pydantic/pydantic/discussions/5853
-class Dataset(BaseModel):
-    context: Literal["https://data.ons.gov.uk/ns#"] = Field(alias="@context")
+class Edition(BaseModel):
     id: str = Field(alias="@id")
-    type: Literal["dcat:DatasetSeries"] = Field(alias="@type")
+    type: Literal["dcat:Dataset"] = Field(alias="@type")
+    in_series: str
     identifier: str
     title: str = Field(max_length=90)
     summary: str = Field(max_length=200)
     description: str = Field(max_length=250)
-    release_date: str = Field(
-        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$"
-    )
-    next_release: str = Field(
-        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$",
-    )
     publisher: str
     creator: str
     contact_point: ContactPoint
-    theme: Union[str, List[str]]
+    topics: Union[str, List[str]]
     frequency: Frequency
     keywords: list[str]
     licence: str
+    issued: str = Field(
+        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$"
+    )
+    modified: str = Field(
+        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$"
+    )
     spatial_coverage: str = Field(pattern=r"^[EJKLMNSW]{1}\d{8}$")
-    temporal_coverage: PeriodOfTime
+    temporal_coverage: str
```

Comment:
Is this temporary? `PeriodOfTime` is the correct way of doing things.

Reply:
It's just working to the example of edition I've got, which at the moment is this: https://github.com/GSS-Cogs/idpd-api-poc/blob/aa94d12909eb478e6616e2e1f4fe3b34a3427401/src/store/metadata/stub/content/editions/cpih_2022-01.json#L38

which I freely admit I don't understand. No issues with changing it, but could give me the new values for that field to match PeriodOfTime, would it just need to match the temporal coverage at the dataset level? i.e this: https://github.com/GSS-Cogs/idpd-api-poc/blob/aa94d12909eb478e6616e2e1f4fe3b34a3427401/src/store/metadata/stub/content/datasets.json#L37

Reply:
Temporal coverage is a span of time, temporal resolution is the list of intervals recorded.

Temporal Coverage: 
using iso8601, `2000-01-01/2009-12-31` or using the Period of Time notation in JSON-LD

```json
"temporal_coverage": {
"@type": "PeriodOfTime",
"start": "2000-01-01"
"end": "2009-12-31"
}

Resolution would be both a list of iso8601 values like, `["P1Y", "P1Q"]` for a dataset which has annual and quarterly resolutions (regardless of start date).

Reply:
My fault! I provided an older example. 🙈